src/RenderingLists.jsx

```diff
@@ -0,0 +1,32 @@
+const listForRender = [
+    { value: 3, name: "List item value 3" },
+    { value: 4, name: "List item value 4" },
+    { value: 10, name: "List item value 10" },
+    { value: 13, name: "List item value 13" },
+    { value: 62, name: "List item value 62" },
+    { value: 22, name: "List item value 22" },
+    { value: 47, name: "List item value 47" },
+    { value: 7, name: "List item value 7" },
+    { value: 8, name: "List item value 8" },
+  ];
+  
+  export function RenderingLists() {
+    const listItems = listForRender.map(list =>
+    <li>{list.name}</li>
+    );
+    const valueList = listForRender.filter(list=>list.value !== 0 ).map(list => {
+        return list.value %2 ===0 ?
+        <li> {list.name}</li> 
+        :<div></div>
+  });
```

Comment:
So you need to change this function this way to work correctly

```suggestion
    const valueList = listForRender.filter(list=> list.value %2 ===0 ).map(list => {
        return <li> {list.name}</li> 
  });
```